### PR TITLE
fix for issues in clustering and persistence

### PR DESCRIPTION
**Pull request by vtuson**
**What this PR does / why we need it**:
When fixing pvc templates in a previous pr, we did not map it to the right volume, so nothing was actually persisted. Once I fixed this , I also found that a setting was not correct and was stopping a restarted node from re-joining the cluster (http://www.rabbitmq.com/cluster-formation.html#node-health-checks-and-cleanup).

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/rabbitmq/templates/statefulset.yaml [hunk 1]
```diff
@@ -80,7 +77,7 @@ spec:
           - name: config-volume
             mountPath: /opt/bitnami/rabbitmq/conf
           - name: data
-            mountPath: /bitnami/rabbitmq
+            mountPath: /opt/bitnami/rabbitmq/var/lib/rabbitmq/
```
Comment (tompizmor) [inline stable/rabbitmq/templates/statefulset.yaml:80]: why was this wrong previously? Also, shouldn't be `/bitnami/rabbitmq/var/lib/rabbitmq/` in any case?

Comment (andresbono) [inline stable/rabbitmq/templates/statefulset.yaml:80]: Please update the [README.md#persistence](https://github.com/kubernetes/charts/tree/master/stable/rabbitmq#persistence) to reflect this change.

Comment (vtuson) [reply stable/rabbitmq/templates/statefulset.yaml:80]: we are not using /bitnami at all

Comment (vtuson) [reply stable/rabbitmq/templates/statefulset.yaml:80]: fixed

File: stable/rabbitmq/README.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ $ helm install --name my-release -f values-production.yaml stable/rabbitmq
 
 ## Persistence
 
-The [Bitnami RabbitMQ](https://github.com/bitnami/bitnami-docker-rabbitmq) image stores the RabbitMQ data and configurations at the `/bitnami/rabbitmq` path of the container.
+The [Bitnami RabbitMQ](https://github.com/bitnami/bitnami-docker-rabbitmq) image stores the RabbitMQ data and configurations at the `/opt/bitnami/rabbitmq/var/lib/rabbitmq/` path of the container.
```
Comment (tompizmor) [inline stable/rabbitmq/README.md:113]: now we only persist data but not configurations, right?

File: stable/rabbitmq/templates/statefulset.yaml (hunk 1 above)
Comment (tompizmor) [reply stable/rabbitmq/templates/statefulset.yaml:80]: you're right! I forgot about it

File: stable/rabbitmq/README.md (hunk 2 above)
Comment (vtuson) [reply stable/rabbitmq/README.md:113]: correct, it persist automatic configurations created by the tool and we overright erlang cookie and config on the entry point script

Comment (vtuson) [reply stable/rabbitmq/README.md:113]: so technically still persisted, So I wouldnt change the comment

Comment (tompizmor) [reply stable/rabbitmq/README.md:113]: ok!

**Pull request merged**